cmd/config/config.go

```diff
@@ -31,6 +31,7 @@ type VendorRoutes struct {
 	GetAll        string `mapstructure:"get-all" validate:"required"`
 	GetByLocation string `mapstructure:"get-by-location" validate:"required"`
 	GetByProduct  string `mapstructure:"get-by-product" validate:"required"`
+	Put           string `mapstructure:"put" validate:"required"`
```

Comment:
I dont think we need this, since we can just change the `http.Method` on the routing to `put`

Reply:
Agreed, it would be redundant to have url like `/vendor/put`. Http method `PUT` with url `/vendor` is the best practice.